rules-emerging-threats/2023/Exploits/CVE-2023-36884/win_security_exploit_cve_2023_36884_office_windows_html_rce_share_access_pattern.yml

```diff
@@ -0,0 +1,28 @@
+title: Potential CVE-2023-36884 Exploitation - Share Access
+id: 3df95076-9e78-4e63-accb-16699c3b74f8
+status: experimental
+description: Detects access to a specific share pattern seen being used during exploitation of CVE-2023-36884
+references:
+    - Internal Research
+author: Nasreddine Bencherchali (Nextron Systems)
+date: 2023/07/13
+tags:
+    - attack.command_and_control
+    - cve.2023.36884
+logsource:
+    product: windows
+    service: security
+    definition: 'The advanced audit policy setting "Object Access > Audit File Share" must be configured for Success/Failure'
+detection:
+    selection_eid:
+        EventID: 5140
+    selection_share_name:
+        ShareName|contains: '\\\\MSHTML_C7\\\\'
+        ShareName|re: '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}'
+    selection_share_path:
+        ShareLocalPath|contains: '\\\\MSHTML_C7\\\\'
+        ShareLocalPath|re: '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}'
```

Comment:
Wanna make sure that you want to match a literal `\\MSHTML_C7\\`. My guess is, that the 5140 event does some escaping itself?

Reply:
I just removed all escaping to make it catch all possible cases. I don't have enough information about how people collect it and the EID itself doesn't do the escaping example ` <Data Name="ShareName">\\*\IPC$</Data>`